src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -0,0 +1,226 @@
+         'time_start': 1541545807.658046,
+         '_Test__params': <AvocadoParams *: * ([]),0: / (['//']),1: r ([]),2: u ([]),3: n ([])>,
+         'file_handler': <logging.FileHandler object at 0x7f342537a250>,
+         '_expected_stdout_file': '/home/brian/daos/daos/src/tests/ftest/test_example.py.data/stdout.expected',
+         '_Test__traceback': None,
+         '_Test__runner_queue': <multiprocessing.queues.SimpleQueue object at 0x7f342537a710>,
+         '_Test__fail_reason': None,
+         'workdir': '/var/tmp/avocado_5z0_yb/1-test_example.py_MyTest.test',
+         '_Test__name': '1-test_example.py:MyTest.test',
+         '_Test__status': None,
+         '_Test__logdir': '/home/brian/daos/daos/src/tests/ftest/avocado/job-results/job-2018-11-06T18.10-4d5d5a4/test-results/1-test_example.py:MyTest.test',
+         '_ssh_logfile': '/home/brian/daos/daos/src/tests/ftest/avocado/job-results/job-2018-11-06T18.10-4d5d5a4/test-results/1-test_example.py:MyTest.test/remote.log',
+         'srcdir': '/var/tmp/avocado_5z0_yb/1-test_example.py_MyTest.test/src',
+         'paused_msg': '',
+         '_Test__sysinfo_logger': <avocado.core.sysinfo.SysInfo object at 0x7f34253983d0>,
+         '_Test__outputdir': '/home/brian/daos/daos/src/tests/ftest/avocado/job-results/job-2018-11-06T18.10-4d5d5a4/test-results/1-test_example.py:MyTest.test/data',
+         '_testMethodName': 'test',
+         '_resultForDoCleanups': None,
+         '_type_equality_funcs': {<type 'set'>: 'assertSetEqual',
+         <type 'dict'>: 'assertDictEqual',
+         <type 'unicode'>: 'assertMultiLineEqual',
+         <type 'list'>: 'assertListEqual',
+         <type 'frozenset'>: 'assertSetEqual',
+         <type 'tuple'>: 'assertTupleEqual'},
+         '_logging_handlers': {'avocado.test.stderr': <logging.FileHandler object at 0x7f342431d8d0>,
+         'avocado.test.stdout': <logging.FileHandler object at 0x7f342431d850>,
+         'paramiko': <logging.FileHandler object at 0x7f342431d950>},
+         '_Test__sysinfo_enabled': <avocado.core.sysinfo.SysInfo object at 0x7f34253ccc10>,
+         '_Test__log': <logging.Logger object at 0x7f34303e3ad0>,
+         '_Test__log_warn_used': False,
+         'timeout': None,
+         '_expected_stderr_file': '/home/brian/daos/daos/src/tests/ftest/test_example.py.data/stderr.expected',
+         '_Test__job': <avocado.core.job.Job object at 0x7f34253c4350>}
+    '''
+    # pylint: enable=line-too-long
+
+    def __init__(self, *args, **kwargs):
+        super(Test, self).__init__(*args, **kwargs)
+        # set a default timeout of 1 minute
+        # tests that want longer should set a timeout in their .yaml file
+        # all tests should set a timeout and 60 seconds will enforce it
+        if not self.timeout:
+            self.timeout = 60
+
+        item_list = self._Test__logdir.split('/')
+        for item_num in range(len(item_list)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len